### PR TITLE
[JENKINS-68270] Fix third party weather icons in table view

### PR DESCRIPTION
**Pull request by NotMyFault**
See [JENKINS-68270](https://issues.jenkins.io/browse/JENKINS-68270)

The change proposed calls the corresponding icon size of the view:

https://user-images.githubusercontent.com/13383509/163467000-41732a4a-32b3-4fb9-ad24-4b6b5e58e753.mp4

I'm unsure why this was set to `icon-lg` before, setting it to `icon-lg` again fixes it in the same way like my approach.

### Proposed changelog entries

* Fix third party weather icons in the table view (regression in 2.341).

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [X] (If applicable) Jira issue is well described
- [X] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate. 
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

### CODE REVIEW

**Review by basil — APPROVED** [review]
Confirmed that this resolves [JENKINS-68270](https://issues.jenkins.io/browse/JENKINS-68270) for me.

File: core/src/main/resources/lib/hudson/ballColorTd.jelly
Comment (timja) [inline core/src/main/resources/lib/hudson/ballColorTd.jelly:61]: weird that it wasn't using this size (`subIconSizeClass`) before, =/

**Review by timja — APPROVED** [review]

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Pull request merged**
